lib/statement.rb

```diff
@@ -0,0 +1,28 @@
+class Statement
+
+  attr_reader :transactions_list
+
+  def initialize(transactions_list)
+    @transactions_list = transactions_list
+  end
+
+  def generate_statement
+    create_header
+    display_records(transactions_list)
+  end
+
+  private
+
+  def create_header
```

Comment:
It's subtle, but this method has two responsibilities: generating the header, and printing it.  Might be worth refactoring it a little.